File: src/main/java/me/telesphoreo/commands/Messages.java
package me.telesphoreo.commands;

import org.bukkit.ChatColor;
import org.spigotmc.SpigotConfig;

public class Messages
{
    static final String MSG_NO_PERMS = ChatColor.RED + "I'm sorry but you do not have permission to perform this command. Please contact the server administrator if you believe that this is in error.";    public static final String PLAYER_NOT_FOUND = ChatColor.RED + "Player not found";
    static final String PLAYER_ONLY = ChatColor.RED + "This command can only be executed by a player";
    static final String CONSOLE_ONLY = ChatColor.RED + "This command can only be executed by the console";
    static final String NO_MSG = ChatColor.RED + "Please provide a message";
    static final String RESTRICTED_TO_SUPERUSERS = ChatColor.RED + "This command is restricted to superusers";
    static final String UNKNOWN_COMMAND = SpigotConfig.unknownCommandMessage;
}

File: src/com/fusenetworks/fuse/commands/Messages.kt
package com.fusenetworks.fuse.commands

import org.bukkit.ChatColor

object Messages {
    val MSG_NO_PERMS = ChatColor.GOLD.toString() + "Fuse >> You do not have permission to do this"
    val INVALID_SERVER = ChatColor.GOLD.toString() + "Fuse >> You are not allowed to use this command on this server"
    val PLAYER_NOT_FOUND = ChatColor.GOLD.toString() + "Fuse >> Player not found"
    val PLAYER_ONLY = ChatColor.GOLD.toString() + "Fuse >> This command can only be executed by a player"
    val CONSOLE_ONLY = ChatColor.GOLD.toString() + "Fuse >> This command can only be executed by the console"
    val NO_MSG = ChatColor.GOLD.toString() + "Fuse >> Please provide a message"
}
